src/openads/application/OpenAds.js

```diff
@@ -7,8 +7,8 @@ export default class OpenAds {
     this._container = container
   }
 
-  createPage ({id, segmentation, positions}) {
-    return this._container.getInstance({key: 'CreatePageUseCase'}).createPage({id, segmentation, positions})
+  addPosition ({domId, name, source, placement, segmentation, sizes, native}) {
```

Comment:
I would use positionId or just id instead of domId if you are referring to the ID of the position (even it is from domain)

Reply:
This PR is not finished, this kind of error will be fixed soon when I push all my local changes ;)